src/main/resources/templates/application/client/address-client-details.html

```diff
@@ -0,0 +1,268 @@
+                            <p class="govuk-body">
+                                Address line 1
+                            </p>
+                        </div>
+                        <div class="govuk-grid-column-one-half">
+                            <p class="govuk-body">
+                                Not applicable
+                            </p>
+                        </div>
+                    </div>
+                    <div class="govuk-grid-row">
+                        <div class="govuk-grid-column-one-half">
+                            <p class="govuk-body">
+                                Address line 2
+                            </p>
+                        </div>
+                        <div class="govuk-grid-column-one-half">
+                            <p class="govuk-body">
+                                Not applicable
+                            </p>
+                        </div>
+                    </div>
+                    <div class="govuk-grid-row">
+                        <div class="govuk-grid-column-one-half">
+                            <p class="govuk-body">
+                                City / Town
+                            </p>
+                        </div>
+                        <div class="govuk-grid-column-one-half">
+                            <p class="govuk-body">
+                                Not applicable
+                            </p>
+                        </div>
+                    </div>
+                    <div class="govuk-grid-row">
+                        <div class="govuk-grid-column-one-half">
+                            <p class="govuk-body">
+                                Country
+                            </p>
+                        </div>
+                        <div class="govuk-grid-column-one-half">
+                            <p class="govuk-body">
+                                Not applicable
+                            </p>
+                        </div>
+                    </div>
+                </div>
+            </div>
+
+            <div class="govuk-grid-row" th:unless="${clientDetails.vulnerableClient}">
+                <div class="govuk-grid-column-two-thirds">
+                    <h2 class="govuk-heading-m">
+                        Main address
+                    </h2>
+                    <div class="govuk-form-group">
+                        <fieldset class="govuk-fieldset">
+                            <legend class="govuk-fieldset__legend">
+                                <h2 class="govuk-fieldset__heading">
+                                    Is your client considered to be homeless and/or do they have no permanent residence?
+                                </h2>
+                                <div id="noFixedAbode-hint" class="govuk-hint">
+                                    If checked then do not enter any address details below and click 'Next'
```

Comment:
Change wording of hint now that we're using radio buttons?
If selecting 'Yes' then do not enter....

Reply:
Amended wording